odoo/addons/base/models/ir_http.py

```diff
@@ -426,7 +426,7 @@ def _binary_record_content(
         return status, content, filename, mimetype, filehash
 
     def _binary_set_headers(self, status, content, filename, mimetype, unique, filehash=None, download=False):
-        headers = [('Content-Type', mimetype), ('X-Content-Type-Options', 'nosniff')]
+        headers = [('Content-Type', mimetype)]
```

Comment:
I don't think it should be removed from there, would be much safer to keep it in both places probably.